sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,2065 @@
+- Added `-fsycl-fp64-conv-emu` command line option which allows to enable
+  partial (only conversion operations are supported) emulation of `double` data
+  type. This mode is only supported by Intel GPUs. intel/llvm#13912
+- Introduced `__PTX_VERSION__` macro that corresponds to the PTX version used
+  when compiling NVPTX. intel/llvm#14621
+
+commit 5b3e7c8f60f0c66ec92f2a19b43ec147d40bd5ed
+    [SYCL][New offload driver][LLVM-SPIRV] Send all translator options to linker wrapper (#13394)
+commit cd24d808382598565f9ec3d9e8faf3e83bfa9aa4
+    [Driver][SYCL] Pass full set of sycl-post-link options to linker wrapper (#13648)
+commit ece73ad61b49eaf9ecb6e2060e5f20e09e26def6
+    [NewOffloadModel][SYCL DeviceLib] Generate SYCL device library objects using new offload model (#13579)
+commit 90c659e4a04e0c74a0eb99d4c72d79c8bc0f783e
+    [Driver][SYCL][NewOffloadModel] Hook up -fsycl-device-only behaviors (#13672)
+commit 91b840c2c44cbc10149f2c77aa4094d6c73bd73f
+    [Driver][SYCL][NewOffloadModel] Hook up -fsycl-device-obj support (#13688)
+commit a8609a5925a3fcb2bd85636702556d15ae5574f4
+    [New offload][llc] Pass -relocation-model=pic option to llc when building shared libraries (#13687)
+commit 16007fa8be4292159f0b19e5fb911b90e3f84aa4
+    [SYCL][Device libs][New offload] Add missing fallback SYCL device library files (#13869)
+commit 5ddc6881a4f5c2ee5f0ccbbd873e57a62bceb30d
+    [Driver][SYCL][NewOffloadModel] Improve arch association for device (#13898)
+commit 7439fb46f1469cf401d89bf203f91ea22bc7ee57
+    [Driver][SYCL][NewOffloadModel] Hook up options for the offload-wrapper (#14001)
+commit 3aee3dbc23d2981c049fdfa6b12b7f759062326d
+    [Driver][SYCL][NewOffload] Update option passing for packager and AOT (#14066)
+commit c2cdfccf4aa90a6da35826c55815f680db3b3805
+    [New offload driver][sycl-post-link] Move sycl-post-link target specific options generation to linker wrapper (#14101)
+commit 934b46f2fff602bcbe7c99c13a1dd7ed6955ae4f
+    [Driver][SYCL][NewOffload] Fix duplication of device targets (#14143)
+commit 6ecce4fdab98b31a897af23444ca4d636af89862
+    [New offload driver][Device lib] Add SYCL device library files for all targets (#14102)
+commit f9fd95ec6c2aa4b77b0503ae1a7a82d6747df105
+    [Driver][SYCL][NewOffloadModel] Incorporate -device settings for GPU (#14151)
+commit 9691782beff5456c297063223ff831d54a8cd624
+    [SYCL][NFC][New offload model][llvm-spirv] Refactor llvm-spirv options generation for enabling correct use under new offload model (#14253)
+commit 3e474e050206e234759115a6442cdd5fb084d3f6
+    [New offload model] Cleanup the way sycl-post-link options are generated (#14177)
+commit fe2b47f08bee73be8bd978c71f3946852e94d790
+    [SYCL][AOT][New offload model] Add AOT support in clang-linker-wrapper for Intel CPUs/GPUs (#14252)
+commit 1c13e6f5e6bae9df42b483852c60631609422043
+    [SYCL][ClangLinkerWrapper] Unconditionally pass -properties to sycl-post-link (#14541)
```

Comment:
@asudarsa, hello, can you please help to formulate in a few short sentences - similar to sentences above - what was done in these commits for New Offloading Model? Only for changes which our users should know / visible to our users.

Reply:
Thanks for working on this Alexey and Dmitry. I will provide a short write-up before EOD today. Thanks

Reply:
Hi @dm-vodopyanov 

Here is a snippet that tries t summarize all my changes above.

`
SYCL offloading is currently performed using the 'old' offloading model where driver creates a static toolchain and makes a sequence of predetermined calls to several clang/llvm tools. In the new offloading model, linking and post-linking of device code is performed inside a dedicated tool - 'clang-linker-wrapper'. New offloading model can be enabled using '--offload-new-driver' option. Following are code changes made in the current release towards adding the new offloading model support for SYCL Old offloading model is currently the default for SYCL offloading:

1. Add AOT support in clang-linker-wrapper for Intel CPUs/GPUs (#14252)
2. Fix up option passing from driver to clang-linker-wrapper for several tools that are called inside clang-linker-wrapper. (#13394, #13648, #13687, #14001, #14066, #14101, #14151, #14253, #14177, #14541) 
3. Add support for some user facing SYCL options in the driver for new offloading model (#13672, #13688)
4. Generate SYCL device library objects using new offload model (#13579, #13869, #14102)
`

Please let me know if any modifications needed.

Thanks